docs/current/guides/snippets/build-local-images/export/main.py

```diff
@@ -0,0 +1,25 @@
+mport sys
+
+import anyio
+import dagger
+
+async def main():
+    # initialize Dagger client
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+
+        # use NGINX container
+        # add new webserver index page
+        c = (
```

Comment:
Nit: using single letter (`c`) is only convention in Go. In other languages it's better to be more descriptive. `ctr` for a `Container` is the shortest we've been using.

Reply:
Done